pom.xml

```diff
@@ -2190,6 +2190,7 @@
                 <spark.version>3.1.3</spark.version>
                 <spark.binary.version>3.1</spark.binary.version>
                 <delta.version>1.0.1</delta.version>
+                <iceberg.version>1.3.1</iceberg.version>
```

Comment:
Iceberg 1.4.0 removes support for spark 3.1

Reply:
let's document it in the comment

"Iceberg removed the support for Spark 3.1 in apache/iceberg#xxxx since 1.4.0"

Reply:
Iceberg removed the support for Spark 3.1 in apache/iceberg#8661 since 1.4.0